src/sentry/snuba/metrics/extraction.py

```diff
@@ -1162,7 +1162,7 @@ def _query_str_for_hash(self) -> str:
     @cached_property
     def query_hash(self) -> str:
         str_to_hash = self._query_str_for_hash
-        hash = hashlib.shake_128(bytes(str_to_hash, encoding="ascii")).hexdigest(4)
+        hash = hashlib.shake_128(bytes(str_to_hash, encoding="utf-8")).hexdigest(4)
```

Comment:
The comment is gone but @vartec pointed out that UTF-8 is a super set of ASCII.
If this was not the case, we would have had a lot of tests failing from query hashes changing.